app/views/businessactivities/expected_amls_turnover.scala.html

```diff
@@ -31,13 +31,21 @@
     heading = header
 ) {
 
-        <ul class="list list-bullet">
-            @bmBusinessActivities.map {activity =>
-              @activity.split('|').map {a =>
-                <li>@a</li>
-              }
+    @bmModel.activities.map { a =>
+        @if(a.businessActivities.size > 1) {
+            <p>@Messages("businessactivities.confirm-activities.subtitle")</p>
+
+            <ul class="list list-bullet">
+            @businessTypes.map { activity =>
+                @activity.split('|').map { a =>
+                    <li>@if(a.charAt(0) == 'a' || a.charAt(0) == 'e') { an @a } else { a @a}</li>
+                }
             }
-        </ul>
+            </ul>
+        } else {
+            <p>@Messages("businessactivities.confirm-activities.subtitle_2") @businessTypes.map { a => @if(a.charAt(0) == 'a' || a.charAt(0) == 'e') {an @a } else { a @a}} </p>
```

Comment:
The functionality in here could be pulled out. Also if it was refactored, it may be easier to understand.

Reply:
How can this be pulled out?